airbyte-webapp/src/components/connection/DestinationStreamNamesModal/DestinationStreamNamesModal.module.scss

```diff
@@ -1,29 +1,29 @@
-@use "scss/colors";
-@use "scss/variables";
-
-.content {
-  display: flex;
-  flex-direction: column;
-  margin: variables.$spacing-xl;
-}
-
-.description {
-  color: colors.$grey;
-  margin-bottom: variables.$spacing-xl;
-}
-
-.radioButton + .radioButton {
-  margin-top: variables.$spacing-xl;
-}
-
-.input {
-  position: relative;
-  margin: variables.$spacing-xl 0 variables.$spacing-xl variables.$spacing-xl;
-
-  .errorMessage {
-    position: absolute;
-    bottom: -16px;
-    left: 0;
-    color: colors.$red-600;
-  }
-}
+@use "scss/colors";
+@use "scss/variables";
+
+.content {
+  display: flex;
+  flex-direction: column;
+  margin: variables.$spacing-xl;
+}
+
+.description {
+  color: colors.$grey;
+  margin-bottom: variables.$spacing-xl;
+}
+
+.radioButton + .radioButton {
+  margin-top: variables.$spacing-xl;
+}
+
+.input {
+  position: relative;
+  margin: variables.$spacing-xl 0 variables.$spacing-xl variables.$spacing-xl;
+
+  .errorMessage {
+    position: absolute;
+    bottom: -16px;
+    left: 0;
+    color: colors.$red-600;
+  }
+}
```

Comment:
Not sure why this is flagged as changed.

Reply:
Me neither, it shouldnt be in the PR...

Reply:
Maybe try changing the base branch away and back to master? Or merging in master again?